gui/wxpython/dbmgr/base.py

```diff
@@ -1678,7 +1678,7 @@ def OnDataItemAdd(self, event):
                             values[i] = int(float(values[i]))
                         elif tlist.columns[columnName[i]]["ctype"] == float:
                             values[i] = float(values[i])
-                    except:
+                    except Exception:
```

Comment:
I'm not sure what you mean here. The exception _Exception_ is not raised here. How this is different from the other cases?

Reply:
The problem is not flake8 W605, but pylint W0716 (broad-exception-raised). Just wondering if you have any suggestions for something different. Keeping ```except Exception:``` here.

Reply:
Yes, `except Exception` causes Pylint's [broad-exception-caught](https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/broad-exception-caught.html), but that's the issue everywhere, not just here, that was my point (and that its broad-exception-caught not broad-exception-raised).

Anyway, I agree that we should be addressing Pylint exceptions as well. There is no point in replacing all `except:` by `except Exception:` because we just exchange one issue for another, but we can make use of the fact that we can resolve the Pylint issue later and get the more clear cases now and leave the difficult cases for later.

For this specific line, my whole issue was just if you point out this as something special, just an example, or something you want to address sooner than the other cases.

Reply:
This is just a sample. You mentioned previously that "except Exception" will silence flake8, but Pylint would have a problem with it, so fixing simple ones has been objective. Just wondering if we should deal with this now or later. I guess we'll address this when we try to fix Pylint problems.

Reply:
If it is difficult to figure out, let's deal with that later.